vcd/resource_vcd_independent_disk.go

```diff
@@ -12,13 +15,16 @@ import (
 	"github.com/vmware/go-vcloud-director/v2/types/v56"
 )
 
+const globalIndependentDiskLockKey = "globalIndependentDiskLockKey"
```

Comment:
Please add a comment about the purpose of this const.

Reply:
added